### ISSUE TITLE
데모 시스템에서 test_a 와 test_b 를 업로드 했을때 성능 패턴이 다르게 개선

### ISSUE THREAD

**Issue by hyu-leeky** [ddps-lab/profet#2]
현재 2개 파일을 다운 받아 실행해보면 패턴이 비슷함.
서로 다른 패턴을 보여주는 샘플 파일을 올리면 좋을것 같음

<img width="830" alt="image" src="https://user-images.githubusercontent.com/17699616/166621274-bd39ec03-a324-4744-93e9-6501acabb7fa.png">

<img width="793" alt="image" src="https://user-images.githubusercontent.com/17699616/166621288-d6f1651c-7ca9-4377-862f-110247c3fd4a.png">

**Comment by james-sungjae-lee** [comment]
네 교수님, 우선 패턴이 비슷한 이유는 의도한 것으로, 두 개의 테스트 케이스가 배치사이즈 이외에는 동일한 워크로드이기 때문입니다.
그렇게 설정한 이유는 해당 결과를 PROFET Scaler Prediction 에서 사용하기 위함입니다.
그러므로 추가적으로 새로운 패턴을 보이는 샘플 파일을 보고자 한다면, Test Data C 로 지정하여 생성하는 것이 맞다고 생각됩니다.
해당 작업은 하루 정도의 오버헤드가 생길 것 같아, 5월 6일까지 마감인 졸업논문 작업을 어느정도 완료하고 진행하겠습니다.

**Comment by hyu-leeky** [comment]
응 성재야. 하나의 케이스를 더 추가하는것도 좋아보인다. 졸업 논문이랑 영어 시험등 우선순위 높은 일들을 다 해결하고 진행하자. 이 건은 다른 사람이 진행해도 괜찮을것 같기도 하고.
